api/generic-handlers.go

```diff
@@ -191,10 +195,23 @@ type RequestIdHandler struct {
 }
 
 func (h RequestIdHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	if helper.CONFIG.OpentracingEnabled {
+		tracer, closer, err := tracing.Init("Yig")
```

Comment:
> The constructor function for Jaeger Tracer returns the tracer itself and an io.Closer instance. It is recommended to structure your main() so that it calls the Close() function on the closer before exiting

https://github.com/jaegertracing/jaeger-client-go

Reply:
I have no idea and I'll test the performance

Reply:
O(1) vs O(n)